storage_devices/disk.py

```diff
@@ -269,15 +276,21 @@ def get_pci_address(cls, device_id):
 
         return pci_address[-1]
 
+    @classmethod
+    def identify(cls, device_name: str) -> bool:
+        output = TestRun.executor.run(
+            f"realpath /sys/block/{device_name}/device/driver | grep nvme"
+        )
+        return output.exit_code == 0
+
 
 class VirtioDisk(Disk):
     plug_all_command = "echo 1 > /sys/bus/pci/rescan"
 
     def __init__(self, path, disk_type, serial_number, block_size):
         Disk.__init__(self, path, disk_type, serial_number, block_size)
         self.plug_command = VirtioDisk.plug_all_command
```

Comment:
`unplug()` should be implemented as a method. And if it's impossible to plugin just one disk, tests should call `plug_all()`, because otherwise the side effect might be quite confusing